courses/fundamentals_of_ada/adv_120_advanced_privacy.rst

```diff
@@ -155,7 +148,9 @@ Tagged
          type T3 is new T2 with null record;
       end P;
 
-* Primitives can be either public or private, except when they have to be derived (constructor functions or abstract subprograms)
+* Primitives can be either public or private
+
+    + Except when they **have** to be derived (constructor functions or abstract subprograms)
```

Comment:
Specify that in this case they have to be public (example maybe?)

Reply:
Ok I added a full slide on the subject of private primitive, LMK if that's ok